temporalnexus/operation.go

```diff
@@ -285,6 +285,97 @@ func (o *workflowRunOperation[I, O]) Start(
 	}, nil
 }
 
+type WorkflowSignalWithStartOperationInput[W, S any] struct {
+	WorkflowOptions client.StartWorkflowOptions
+	WorkflowArg     W
+	SignalName      string
+	SignalArg       S
+}
+
+type WorkflowSignalWithStartOperationOptions[I, W, S, O any] struct {
+	Name     string
+	Workflow func(workflow.Context, W) (O, error)
+	GetInput func(context.Context, I, nexus.StartOperationOptions) (WorkflowSignalWithStartOperationInput[W, S], error)
+	Handler  func(context.Context, I, nexus.StartOperationOptions) (WorkflowHandle[O], error)
```

Comment:
I wonder if this would be easy for users to leverage. I'm not sure returning a `WorkflowHandle[0]` is desirable here, maybe we need a different return type.
We need a `Handler` function for when users have non-standard workflow signatures, it could leverage a helper method to attach the relevant fields to the signal-with-start request.

Here's an example of what this might look like
```go
func SignalWithStartUntypedWorkflow[R any](
	ctx context.Context,
	nexusOptions nexus.StartOperationOptions,
	startWorkflowOptions client.StartWorkflowOptions,
	signalName string,
	signalArg any,
	workflow any,
	workflowArgs ...any,
) error {
```